### ISSUE TITLE
Lion Pet Dmg massively inconsistent in rvr, and often doing 0 dmg. This is not affected by range.

### ISSUE THREAD

**Issue by iantheace** [WarEmu/WarBugs#17986]
I have done some tests (about 5 fights) and white lion pet dmg is literally all over the place in RVR, even hitting the <same> target. It is perfectly consistent on target dummies.

I posted other pics in discord earlier, of the pet hitting for 20 dmg on a witch elf.... See extract below

![PetDmgInconsistentpng](https://user-images.githubusercontent.com/11569450/97201387-4c7be800-17aa-11eb-9775-88bd1281aa45.png)

pet hits one time (for 108 dmg with basic attack). then hits another time (for 19 dmg with basic attack).

Here is a small part of the log, this happens constantly.

**Comment by CrimsonFenris** [comment]
Total damage on pet's auto-attacks remains the same, just being detaunted byt the WE I guess (going from 40 to 19) .

Last attack looks like the WE got out of guard range, so she's taking full dmg (guard only apply to remaining dmg, after all previous mitigation from toughness and armor).

My guess is ; WE is guarded when hit by your pet, she AoE detaunt (pet doing half dmg), then she got outside of guard range (Pet doing full dmg).

Many factors are influencing output damage, wich are obviously not reflected in combat log (like detaunt and guard).

**Comment by iantheace** [comment]
> Total damage on pet's auto-attacks remains the same, just being detaunted byt the WE I guess (going from 40 to 19, both are 188 total damage) .
> 
> Last attack looks like the WE got out of guard range, so she's taking full dmg (guard only apply to remaining dmg, after all previous mitigation from toughness and armor).
> 
> My guess is ; WE is guarded when hit by your pet, she AoE detaunt (pet doing half dmg), then she got outside of guard range (Pet doing full dmg).
> 
> Many factors are influencing output damage, wich are obviously not reflected in combat log (like detaunt and guard).

This is a solo witch elf (no guard), who had not detaunted (obviously) as she is hitting me (would isntantly remove the detaunt).

**Comment by dalen** [comment]
If the pet is far away from you, the damage is reduced. Can't really tell the pet distance from that combat log

**Comment by iantheace** [comment]
> If the pet is far away from you, the damage is reduced. Can't really tell the pet distance from that combat log

I am using coordinated strike which requires you to be adjacent to the target 😀

**Comment by iantheace** [comment]
I think though Dalen a lot of these issues are due to the server thinking the pet is far away when it isn’t. It often glitches into the middle of nowhere and it disappears too. 

I guess this is much more of a problem with pets that move. But either this needs fixing, or the pet debuffs need removing

**Comment by dalen** [comment]
The damage will also vary a lot depending on which training you used. Also not clear if that was switched during that combat log.

You can test the damage out on a training construct as well now in the city if you want to try an armoured target. But from this combat log it is a bit unclear what bug you are reporting.

**Comment by dalen** [comment]
All the lion auto attacks are 188 damage non-crit, except the last one which is 294 damage, I guess you used Leonine Frenzy then.

**Comment by CrimsonFenris** [comment]
@dalen Judging by the OP screen Leonine frenzy have not been specced (Whirling axe + Primal Fury does not allow for guardian tree).

In my bug report (https://github.com/WarEmu/WarBugs/issues/18006) I was using Coordinated strike on nearby target, and damage is less than half the tooltip for the pet wich attacked same target (and obviously was near me).

Training should not impact CS damage, but I think it's related to the server miscalculating distance between pet and character, wich create odd results, both in my case and Iantheace's one.

**Comment by dalen** [comment]
The pet uses its own stats when attacking. So pet training definitely impacts the damage from it.

**Comment by CrimsonFenris** [comment]
Fair enough, but either pet stat contribution for that skill are not high enough, or the tooltip is wrong ; CS should gets your pet almost same damage than yours ; now its around half.

Tooltip is showing 368 damage (x2) for my pet, and I have 1010 STR :

In Trained to Kill (all STR bonus to pet), it deals only 224 (x2),
With Trained to Threaten its 124 (x2) (not STR bonus),
Trained to Hunt 164 (x2) (half STR bonus).

I clearly remember this attack used to get you around the tooltip damage (give or take), but it was around the same damage than our character, so I guess this attack have now unintended low damage for the pet's part, and either toolptip need to be adjusted to reflect that, or stat contribution for the pet should be almost doubled to match tooltip (at least in trained to Kill training wich is the maximum STR bonus for the pet).

**Comment by iantheace** [comment]
> All the lion auto attacks are 188 damage non-crit, except the last one which is 294 damage, I guess you used Leonine Frenzy then.

No, I’m not specced for leonine frenzy, the pet dps is just all over the place attacking anything but a dummy pretty much. I beleive it’s to do with the distance to the pet being randomly miscalculated all the time.

**Comment by iantheace** [comment]
> The pet uses its own stats when attacking. So pet training definitely impacts the damage from it.

Pet buffs and training randomly do and don’t work, (it’s rare they do for me) I supplied a video of the pet doing the same dps with all 3 buffs. 

Honestly don’t know what’s been changed but the pet is basically useless other than the odd occasion where you get lucky and it works and your class isn’t gimped again
